### PR TITLE
Add up/downstream term to the glossary

### PR DESCRIPTION
**Pull request by DaRaFF**
* Add up/downstream term to the glossary
* Introduce alphabetical order

### CODE REVIEW

File: GLOSSARY.md [hunk 1]
```diff
@@ -1,29 +1,38 @@
...
 
 A print publishing system.
+
+## Project
+
+Represents an organisation and consists of users and channels.
+
+## Upstream
+
+*Upstream* is a package/repository which is integrated in another project called *downstream*. As an example: There is an *upstream* project called livingdocs-server(core) which is integrated/used by a *downstream* project livingdocs-server-downstream.
```
Comment (gabrielhase) [inline GLOSSARY.md]: That sounds too generic. We don't need to explain what an upstream in general is but only in the context of our software, so please tell that it is our core software, that it is an npm package and that we use semantic versioning.

Comment (marcbachmann) [reply GLOSSARY.md]: I'd keep the first sentence, replace the `As an example` with `In our case` or something like that.

**Comment by DaRaFF** [comment]
@gabrielhase please check it again. I renamed glossary to dictionary.

File: DICTIONARY.md [hunk 2]
```diff
@@ -1,29 +1,38 @@
...
 
 A print publishing system.
+
+## Project
+
+Represents an organisation and consists of users and channels.
+
+## Upstream
+
+*Upstream* is a package/repository which is integrated in another project called *downstream*. In our case *upstream* means our core editor and server is provided as a npm package (semantic versioning) and can be used by a *downstream* project e.g. `livingdocs-server-downstream`.
```
Comment (gabrielhase) [inline DICTIONARY.md]: I am nitpicking :) please add a hint that the framework does not have a downstream/upstream separation and can not be installed separately.

**Pull request merged**